literature.bib

```diff
@@ -337,6 +337,38 @@ @mastersthesis{Guething23
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2024 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{BRSK:JSS24,
+  author       = {Alexander Boll and
+                  Pooja Rani and
+                  Alexander Schulthei{\ss} and
+                  Timo Kehrer},
+  title        = {{Beyond code: Is there a difference between comments in visual and
+                  textual languages?}},
```

Comment:
capitalization wrong: https://capitalizemytitle.com/

Reply:
done

Reply:
Previously, I used the official capitalization of the Journal. I am aware of the website that you linked.

Reply:
Ok, we never use the official capitalization. ACM also uses lower case titles.